modules/gapi/src/backends/oak/goak_media_adapter.cpp

```diff
@@ -0,0 +1,60 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#include "oak_media_adapter.hpp"
+
+namespace cv {
+namespace gapi {
+namespace oak {
+
+class OAKMediaAdapter::Priv final {
```

Comment:
So far this class has nothing special about OAK memory management (if there is such thing at all).

Can this be replaced with some naive implementation taken from our MediaFrame tests?

Reply:
Except for the specific constructor it's pretty generic. What do you suggest to do here?

Reply:
Nothing, and that's the question